File: Beginner/2936 - How Much Cassava/2936.c
#include <stdio.h>

int main()
{
    int table[5] = {300, 1500, 600, 1000, 150};
    int gramas = 225;

    for(int i = 0; i<5; i++){
        int aux;
        scanf("%i", &aux);
        gramas += table[i]*aux;
    }
    printf("%i\n", gramas);
    return 0;
}

File: Beginner/2936 - How Much Cassava/2936.java
import java.util.Scanner;

class Main {

    public static void main(String[] args) {
        int[] table = {300, 1500, 600, 1000, 150};
        int gramas = 225;
        Scanner scanner = new Scanner(System.in);
        
        for(int i = 0; i<5; i++){
            int aux = scanner.nextInt();
            gramas += table[i]*aux;
        }
        System.out.println(gramas);
        scanner.close();
    }
}